internal/collector/eth_block_timestamp.go

```diff
@@ -0,0 +1,53 @@
+package collector
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type EthBlockTimestamp struct {
+	rpc  *rpc.Client
+	desc *prometheus.Desc
+}
+
+type blockResult struct {
+	Timestamp hexutil.Uint64
+}
+
+func NewEthBlockTimestamp(rpc *rpc.Client) *EthBlockTimestamp {
+	return &EthBlockTimestamp{
+		rpc: rpc,
+		desc: prometheus.NewDesc(
+			"eth_block_timestamp",
```

Comment:
Please also update https://github.com/31z4/ethereum-prometheus-exporter/blob/master/README.md#exported-metrics with metric's name and description.